recipes/pyserini/setup.patch

```diff
@@ -0,0 +1,21 @@
+From 3b31e323ea9752afd61ec8c8c0ec7ebeca89ef21 Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Tue, 18 Jun 2024 13:01:14 +0200
+Subject: [PATCH] Update setup.py
+
+---
+ setup.py | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index 4fb3bba27..9a9964363 100644
+--- a/setup.py
++++ b/setup.py
+@@ -7,7 +7,6 @@
+     requirements = f.read().splitlines()
+ 
+ pyserini_packages = setuptools.find_packages()
+-pyserini_packages.remove('tests')
```

Comment:
This would leave a package named tests in the namespace, no?

Reply:
It raises an error:
```
Processing $SRC_DIR
  Added file://$SRC_DIR to build tracker '/tmp/pip-build-tracker-z5u14kwf'
  Running setup.py (path:$SRC_DIR/setup.py) egg_info for package from file://$SRC_DIR
  Created temporary directory: /tmp/pip-pip-egg-info-suutj552
  Preparing metadata (setup.py): started
  Running command python setup.py egg_info
  Traceback (most recent call last):
    File "<string>", line 2, in <module>
    File "<pip-setuptools-caller>", line 34, in <module>
    File "/home/conda/staged-recipes/build_artifacts/pyserini_1718704178167/work/setup.py", line 10, in <module>
      pyserini_packages.remove('tests')
  ValueError: list.remove(x): x not in list
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output
```
This is why I tested building from Github directly. But then on Github the Java `*.jar` files seem to be missing, so I ended up with fixing the pip package.

Reply:
That code is probably not shipped in the sdist. It should be fine then.